.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         - *js_deps_cache_key
     - run:
         name: Conditional install
-        command: if [ ! -d node_modules ]; then npm install; fi
+        command: if [ ! -d node_modules ]; then npm install --no-save; fi
```

Comment:
After reading docs [npm ci](https://docs.npmjs.com/cli/ci.html) is technically more correct. Sorry, I'll reapprove!